src/components/Camera/style.css

```diff
@@ -16,63 +16,74 @@
 }
 
 .overlayFullScreen {
-  @media (--small-viewport) {
+  /*@media (--small-viewport) {*/
```

Comment:
Video capturing now looks fullscreen on desktop, so `isFullscreen` should now affect all viewports, and only small viewports:

![image](https://user-images.githubusercontent.com/1504258/42717414-17c12910-86d7-11e8-83e3-fefac7f51361.png)

I know `Photo` will also be affected by this, so for now I commented these media queries out until I can confirm `Photo` still needs to be restrained inside a rectangle on larger viewports.